src/main/java/frc/robot/RobotContainer.java

```diff
@@ -98,13 +113,14 @@ private void configureButtonBindings() {
     fourBarClimb.whenHeld(new ClimbRoutine(redArm, yellowArm, tiltArm));
   }
 
+
```

Comment:
Be mindful of how and where you use whitespace. It should be curated to increase the readability of the code by someone else and inconsistent use diminishes that. This line should be deleted.